pkg/components/exporter/exporter_test.go

```diff
@@ -0,0 +1,75 @@
+package exporter
+
+import (
+	"testing"
+
+	ko "github.com/sustainable.computing.io/kepler-operator/pkg/api/v1alpha1"
+	"gotest.tools/v3/assert"
```

Comment:
nit: we should rely on just one assert package. I am okay with any :) but pick one and use that.

Reply:
is there something already chosen? i will use that.